python/tests/res/fm/test_util.py

```diff
@@ -0,0 +1,73 @@
+
+from ecl.util.test import TestAreaContext
+from tests import ResTest
+from res.fm.util import *
+
+import subprocess
+import os
+import sys
+
+class UtilTest(ResTest):
```

Comment:
The test coverage for the similar job in _Everest_ is quite a bit better. Please revise the cases that you are not currently covering here. In particular having multiple input files and richer template files utilising iteration.